apps/omg_watcher/test/exit_processor/persistence_test.exs

```diff
@@ -0,0 +1,240 @@
+# Copyright 2018 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Watcher.ExitProcessor.PersistenceTest do
+  @moduledoc """
+  Test focused on the persistence bits of `OMG.Watcher.ExitProcessor.Core`.
+
+  The aim of this test is to ensure, that whatever state the processor ends up being in will be revived from the DB
+  """
+
+  use ExUnitFixtures
+  use OMG.DB.Case, async: true
+
+  alias OMG.API.DevCrypto
+  alias OMG.API.State.Transaction
+  alias OMG.API.Utxo
+  alias OMG.Watcher.ExitProcessor.Core
+
+  require Utxo
+
+  @eth OMG.API.Crypto.zero_address()
+  @not_eth <<1::size(160)>>
```

Comment:
the same about this

Reply:
this one is not that frequent. I think DRYing this would confuse more (new module under `support` required etc etc) than clarify